src/locale/messages.fr.xlf

```diff
@@ -25,6 +25,20 @@
           <context context-type="sourcefile">src/app/core/components/group-nav/group-nav.component.html</context>
           <context context-type="linenumber">12</context>
         </context-group>
+      </trans-unit><trans-unit id="871322351859596725" datatype="html">
+        <source>Direct Children Only</source><target state="translated">Enfants uniquement</target>
```

Comment:
I feel like we are losing the meaning of "direct" in "Direct Children Only".

Also, it seems strange that in English every word is capitalized and not in French.

Reply:
Capitalization of each word for titles is something which exists in English style and not in French. Anyway, here it is all uppercased ;-).
I find "Enfants directs uniquement" very long and boring in this menu, but maybe "Enfants directs" would be better than "Enfants uniquement".